sensu/files/plugins/memory-metrics.rb

```diff
@@ -68,16 +68,20 @@ def metrics_hash
 
     mem['swapUsed'] = mem['swapTotal'] - mem['swapFree']
     mem['used'] = mem['total'] - mem['free']
-    memory_used = (100.0 * mem['used']) / mem['total']
+    mem['usedWOBuffersCaches'] = mem['used'] - (mem['buffers'] + mem['cached'])
+    mem['freeWOBuffersCaches'] = mem['free'] + (mem['buffers'] + mem['cached'])
+    mem['swapUsedPercentage'] = 100 * mem['swapUsed'] / mem['swapTotal'] if mem['swapTotal'] > 0
```

Comment:
Ok, now I have edited it so that in the case that the memory used by the computer is beyond the threshold, it will output "critical" and then it will output the top 10 processes  in terms of their memory usage, along with the user, PID, %memory used, and a short amount of the command. Example:

critical
USER       PID %MEM COMMAND
qemu     14121  1.4 /usr/libexec/qemu-kvm -name tempest-minying -S -machine
qemu     14187  1.6 /usr/libexec/qemu-kvm -name tempest201603 -S -machine p
qemu     18113  1.7 /usr/libexec/qemu-kvm -name mocorgmail -S -machine pc-i
qemu     32260  1.7 /usr/libexec/qemu-kvm -name moc-redhatrepos -S -machine
qemu     15461  3.0 /usr/libexec/qemu-kvm -name dev-foreman1 -S -machine pc
qemu     11848  3.1 /usr/libexec/qemu-kvm -name kaizen-sensu-admin -S -mach
qemu     15168  4.3 /usr/libexec/qemu-kvm -name moc-ceilometer -S -machine 
qemu     27557  4.5 /usr/libexec/qemu-kvm -name moc-logstash -S -machine pc
qemu     27401  4.5 /usr/libexec/qemu-kvm -name moc-foreman -S -machine pc-
qemu      7126 27.0 /usr/libexec/qemu-kvm -name moc-sensu -S -machine pc-i4